index.html

```diff
@@ -10,5 +10,27 @@
 		<meta name="viewport" content="width=device-width, initial-scale=1"/>
 		<link rel="stylesheet" href="css/style.css">
 	</head>
-	<body></body>
+	<body>
+		<div class="container">

```

Comment:
For Index_V3 I added my slides in, both the vertical container ones and the horizontal container ones.